src/Analysis/Ast/Impl/Extensions/AstExtensions.cs

```diff
@@ -17,16 +17,18 @@
 using System.Diagnostics;
 using System.Linq;
 using Microsoft.Python.Analysis.Analyzer.Expressions;
+using Microsoft.Python.Analysis.Modules;
```

Comment:
This is suspicious. AST extensions should not be needing analysis. We may actually move most of extensions to `Python.Parsing`

Reply:
The trouble here will be getting a matching pairing of AST and file content. It's a shame that we need the AST in order to convert a line number to an offset, only to then go back to the content to split lines.

Reply:
Yeah, but it is called on `module.GetAst()` anyway